x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -99,7 +99,20 @@ field
     ;
 
 fromCommand
-    : FROM fromIdentifier (COMMA fromIdentifier)* metadata?
+    : FROM fromIdentifier (COMMA fromIdentifier)* fromOptions? metadata?
```

Comment:
Yes, I had that thought too (it's just a grammar line more). But had a look at SQL (which doesn't allow this swapping where it could) and thought of how we now request quotes (i.e. no flexibility) and thought I'd keep it inline.

Reply:
I missed this - I'm fine with the order being fixed however I would keep the metadata? first, e.g.:
`FROM index METADATA ... OPTIONS...` since in general I expect OPTIONS to be the last item in a command mainly because there can be a long list of parameters.